chain-abci/src/staking/tx.rs

```diff
@@ -24,9 +24,9 @@ impl StakingTable {
         heap: &mut impl StoreStaking,
         block_time: Timespec,
         max_evidence_age: Timespec,
-        ra_verifier: &EnclaveCertVerifier,
+        mut recent_isv_svn: u16,
```

Comment:
some full-word comment would be useful

Reply:
added a little bit more comment here about isv svn:
https://github.com/crypto-com/chain/pull/1668/files#diff-3cfbc20473df3695f7d44c7c3d8c1508R57